components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -152,13 +152,15 @@ function ConversationList(props: ConversationListProps) {
                   {isAIAssistant && (
                     <div className={styles.turnActions}>
                       <CopyButton onClick={handleCopyText} />
-                      <ContextMenuAssistant
-                        ref={portalRefs}
-                        turnId={id}
-                        isOpen={activeMenuId === id}
-                        onClick={() => showAssistantMenu(id)}
-                        onClose={hideAssistantMenu}
-                      />
+                      {context.currentModel?.options.leoModelOptions && (
```

Comment:
Don't we have a `isLeoModel(model)` function?

Reply:
yea, but it's not globally shared and i know you're not a fan of utils file. 

we could perhaps send this in context as a computed `isCurrentModelLeo`

Reply:
ok, sending it through Context now.